packages/cosmic-swingset/src/kernel-stats.js

```diff
@@ -284,7 +284,7 @@ export function makeInboundQueueMetrics(initialLength) {
 /**
  * @param {object} param0
  * @param {any} param0.controller
- * @param {import('@opentelemetry/api-metrics').Meter} param0.metricMeter
+ * @param {import('@opentelemetry/api').Meter} param0.metricMeter
```

Comment:
this commit is titled "fix(cosmic-swingset): adapt to new @opentelemetry". For the changelog, there is nothing to fix. This is really a fixup to the change that bumped the OpenTelemetry version. Better to squash it into that commit. Even better is isolating the opentelemetry changes (deps and code) from the other commits.